### ISSUE TITLE
Add rolling D-Pad

### ISSUE THREAD

**Issue by Shugabuga** [iGBAEmu/EclipseIssues#11]
Currently, you cannot roll your finger on your screen to trigger another button, or easily hold two buttons at once (like A and B). If this is fulfilled, then you would be able to.

[Relevant Reddit thread](https://www.reddit.com/r/EclipseEmu/comments/7yqyhp/unable_to_load_gba_from_google_drive/dukb4ee/)

**Comment by DavidHouse369** [comment]
This would be a great improvement to the emulator. Currently there are a lot of games that are very tedious to play as they are designed for diagonal movement (eg. Megaman Battle Network), but there are also games that require rapid changes in direction in order to play (eg. Metal Slug).

**Comment by Manny141** [comment]
Yes, it’s kinda difficult to play a lot of retro games with only going in 4 direction such as contra and street fighter

**Comment by Missingeter** [comment]
TBH adding a rolling d-pad would make the whole eclipse experience so much better,  it has been an annoyance in so many games and if we included a rolling d-pad everything would be so much more fun
